[0m2021.07.19 21:48:01 INFO  Started: Metals version 0.10.5 in workspace 'C:\Users\14694\OneDrive\Desktop\vs Code Example' for client Visual Studio Code 1.58.2.[0m
[0m2021.07.19 21:48:01 INFO  time: initialize in 0.48s[0m
[0m2021.07.19 21:48:01 WARN  Build server is not auto-connectable.[0m
[0m2021.07.19 21:48:01 WARN  no build target for: C:\Users\14694\OneDrive\Desktop\vs Code Example\build.sbt[0m
[0m2021.07.19 21:48:01 WARN  no build target for: C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala[0m
Jul 19, 2021 9:48:06 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
[0m2021.07.19 21:48:06 INFO  running 'C:\Users\14694\.jabba\jdk\adopt@1.8.0-292\bin\java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -Dsbt.version=1.5.4 -jar C:\Users\14694\AppData\Local\Temp\metals1982432513548429515\sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'[0m
[0m2021.07.19 21:48:08 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.07.19 21:48:08 INFO  [info] Updated file C:\Users\14694\OneDrive\Desktop\vs Code Example\project\build.properties: set sbt.version to 1.5.4[0m
[0m2021.07.19 21:48:08 INFO  [info] welcome to sbt 1.5.4 (AdoptOpenJDK Java 1.8.0_292)[0m
[0m2021.07.19 21:48:09 INFO  [info] loading settings for project global-plugins from build.sbt ...[0m
[0m2021.07.19 21:48:09 INFO  [info] loading global plugins from C:\Users\14694\.sbt\1.0\plugins[0m
[0m2021.07.19 21:48:10 INFO  time: code lens generation in 8.11s[0m
[0m2021.07.19 21:48:13 INFO  [info] loading settings for project vs-code-example-build-build from metals.sbt ...[0m
[0m2021.07.19 21:48:13 INFO  [info] loading project definition from C:\Users\14694\OneDrive\Desktop\vs Code Example\project\project[0m
[0m2021.07.19 21:48:20 INFO  [info] loading settings for project vs-code-example-build from metals.sbt ...[0m
[0m2021.07.19 21:48:20 INFO  [info] loading project definition from C:\Users\14694\OneDrive\Desktop\vs Code Example\project[0m
[0m2021.07.19 21:48:22 INFO  [success] Generated .bloop\vs-code-example-build.json[0m
[0m2021.07.19 21:48:22 INFO  [success] Total time: 2 s, completed Jul 19, 2021 9:48:23 PM[0m
[0m2021.07.19 21:48:24 INFO  [info] loading settings for project vs-code-example from build.sbt ...[0m
[0m2021.07.19 21:48:24 INFO  [info] set current project to HelloWorldApp (in build file:/C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/)[0m
[0m2021.07.19 21:48:30 INFO  [success] Generated .bloop\vs-code-example.json[0m
[0m2021.07.19 21:48:30 INFO  [success] Generated .bloop\vs-code-example-test.json[0m
[0m2021.07.19 21:48:30 INFO  [success] Total time: 6 s, completed Jul 19, 2021 9:48:30 PM[0m
[0m2021.07.19 21:48:30 INFO  sbt bloopInstall exit: 0[0m
[0m2021.07.19 21:48:30 INFO  time: ran 'sbt bloopInstall' in 24s[0m
[0m2021.07.19 21:48:30 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 20788'...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
Waiting for the bsp connection to come up...
No server running at 127.0.0.1:8212, let's fire one...
Resolving ch.epfl.scala:bloop-frontend_2.12:1.4.8-19-4d9f966b...
Starting bloop server at 127.0.0.1:8212...
Attempting a connection to the server...
[0m[32m[D][0m Loading 2 projects from 'c:\Users\14694\OneDrive\Desktop\vs Code Example\.bloop'...
[0m[32m[D][0m Loading project from 'c:\Users\14694\OneDrive\Desktop\vs Code Example\.bloop\vs-code-example.json'
[0m[32m[D][0m Loading project from 'c:\Users\14694\OneDrive\Desktop\vs Code Example\.bloop\vs-code-example-test.json'
[0m[32m[D][0m Cache miss for scala instance org.scala-lang:scala-compiler:2.11.12.
[0m[32m[D][0m   => C:\Users\14694\AppData\Local\Coursier\Cache\v1\https\repo1.maven.org\maven2\jline\jline\2.14.3\jline-2.14.3.jar
[0m[32m[D][0m   => C:\Users\14694\AppData\Local\Coursier\Cache\v1\https\repo1.maven.org\maven2\org\scala-lang\modules\scala-parser-combinators_2.11\1.0.4\scala-parser-combinators_2.11-1.0.4.jar
[0m[32m[D][0m   => C:\Users\14694\AppData\Local\Coursier\Cache\v1\https\repo1.maven.org\maven2\org\scala-lang\modules\scala-xml_2.11\1.0.5\scala-xml_2.11-1.0.5.jar
[0m[32m[D][0m   => C:\Users\14694\AppData\Local\Coursier\Cache\v1\https\repo1.maven.org\maven2\org\scala-lang\scala-compiler\2.11.12\scala-compiler-2.11.12.jar
[0m[32m[D][0m   => C:\Users\14694\AppData\Local\Coursier\Cache\v1\https\repo1.maven.org\maven2\org\scala-lang\scala-library\2.11.12\scala-library-2.11.12.jar
[0m[32m[D][0m   => C:\Users\14694\AppData\Local\Coursier\Cache\v1\https\repo1.maven.org\maven2\org\scala-lang\scala-reflect\2.11.12\scala-reflect-2.11.12.jar
[0m[32m[D][0m Missing analysis file for project 'vs-code-example'
[0m[32m[D][0m Missing analysis file for project 'vs-code-example-test'
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:20788...
The server is listening for incoming connections at tcp://127.0.0.1:20788...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.19 21:48:45 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.19 21:48:45 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 19431'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:19431...
The server is listening for incoming connections at tcp://127.0.0.1:19431...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.19 21:48:49 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.19 21:48:49 INFO  time: Connected to build server in 18s[0m
[0m2021.07.19 21:48:49 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.19 21:48:49 INFO  time: Imported build in 0.25s[0m
[0m2021.07.19 21:48:55 INFO  time: indexed workspace in 6.48s[0m
[0m2021.07.19 21:48:57 INFO  compiling vs-code-example (2 scala sources)[0m
[0m2021.07.19 21:49:01 INFO  time: compiled vs-code-example in 4.77s[0m
Jul 19, 2021 9:50:30 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jul 19, 2021 9:50:33 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassNotFoundException: proj0
java.util.concurrent.CompletionException: java.lang.ClassNotFoundException: proj0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: proj0
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.$anonfun$findClassAndBuildTarget$1(BuildTargetClassesFinder.scala:107)
	at scala.Option.fold(Option.scala:251)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findClassAndBuildTarget(BuildTargetClassesFinder.scala:108)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findMainClassAndItsBuildTarget(BuildTargetClassesFinder.scala:36)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$1(DebugProvider.scala:328)
	at scala.meta.internal.metals.debug.DebugProvider$$anonfun$withRebuildRetry$1.$anonfun$applyOrElse$5(DebugProvider.scala:560)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 21:51:01 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 21:51:01 INFO  time: compiled vs-code-example in 0.95s[0m
[0m2021.07.19 21:51:11 INFO  Listening for transport dt_socket at address: 53631[0m
[0m2021.07.19 21:51:12 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 21:51:16 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 21:51:16 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 21:51:16 INFO  Trying to attach to remote debuggee VM 127.0.0.1:53631 .[0m
[0m2021.07.19 21:51:16 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.19 21:51:16 INFO  Hello, world![0m
[0m2021.07.19 21:51:16 INFO  Canceling debug proxy for [proj0][0m
Jul 19, 2021 9:51:17 PM org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer fireError
SEVERE: java.net.SocketException: Socket closed
org.eclipse.lsp4j.jsonrpc.JsonRpcException: java.net.SocketException: Socket closed
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:72)
	at scala.meta.internal.metals.debug.SocketEndpoint.consume(SocketEndpoint.scala:22)
	at scala.meta.internal.metals.debug.MessageIdAdapter.consume(MessageIdAdapter.scala:43)
	at scala.meta.internal.metals.debug.ServerAdapter.send(ServerAdapter.scala:30)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$handleClientMessage$1(DebugProxy.scala:96)
	at scala.meta.internal.metals.debug.MessageIdAdapter.$anonfun$listen$1(MessageIdAdapter.scala:57)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at scala.meta.internal.metals.debug.SocketEndpoint.listen(SocketEndpoint.scala:26)
	at scala.meta.internal.metals.debug.MessageIdAdapter.listen(MessageIdAdapter.scala:47)
	at scala.meta.internal.metals.debug.DebugProxy.$anonfun$listenToClient$1(DebugProxy.scala:61)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.SocketException: Socket closed
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:118)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:155)
	at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)
	at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:140)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageConsumer.consume(StreamMessageConsumer.java:69)
	... 21 more

something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 120, 120, 133)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 146, 146, 159)
[0m2021.07.19 22:03:59 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 15m 14.73s)[0m
[0m2021.07.19 22:03:59 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:03:59 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:03:59 INFO  time: compiled vs-code-example in 0.55s[0m
[0m2021.07.19 22:04:02 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:04:02 INFO  time: compiled vs-code-example in 0.38s[0m
Jul 19, 2021 10:04:05 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:05:23 INFO  compiling vs-code-example (1 scala source)[0m
Jul 19, 2021 10:05:23 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:05:23 INFO  time: compiled vs-code-example in 0.4s[0m
Jul 19, 2021 10:05:55 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:05:55 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:05:55 INFO  time: compiled vs-code-example in 0.4s[0m
Jul 19, 2021 10:11:51 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:11:51 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:11:51 INFO  time: compiled vs-code-example in 0.52s[0m
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
Jul 19, 2021 10:12:45 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:12:45 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:12:45 INFO  time: compiled vs-code-example in 0.42s[0m
[0m2021.07.19 22:13:05 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:13:05 INFO  time: compiled vs-code-example in 0.3s[0m
Jul 19, 2021 10:13:22 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 755
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[<error>]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 82)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[<error>]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 75)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[<error>]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 76)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[<error>]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 78)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
Jul 19, 2021 10:14:15 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:14:16 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:14:16 INFO  time: compiled vs-code-example in 0.37s[0m
[0m2021.07.19 22:14:53 INFO  compiling vs-code-example (1 scala source)[0m
Jul 19, 2021 10:14:53 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:14:53 INFO  time: compiled vs-code-example in 0.32s[0m
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
Jul 19, 2021 10:19:19 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1024
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/Hello_World.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/Hello_World.scala, 41, 41, 54)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/Hello_World.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/Hello_World.scala, 41, 41, 54)
Jul 19, 2021 10:19:27 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1036
Jul 19, 2021 10:20:54 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:20:54 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:20:54 INFO  time: compiled vs-code-example in 0.41s[0m
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 70, 70, 83)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 70, 70, 83)
Jul 19, 2021 10:25:49 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1304
Jul 19, 2021 10:25:57 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 1315
[0m2021.07.19 22:26:15 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:12:24: stale bloop error: value e2e is not a member of Int
    var credit_score = readInt()e2e
                       ^^^^^^^^^^^^[0m
[0m2021.07.19 22:26:15 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:31: stale bloop error: not found: value name
     var displyName = println(name)
                              ^^^^[0m
[0m2021.07.19 22:26:15 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:23:28: stale bloop error: not found: value age
   var displyAge = println(age)
                           ^^^[0m
[0m2021.07.19 22:26:15 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:12:24: stale bloop error: value e2e is not a member of Int
    var credit_score = readInt()e2e
                       ^^^^^^^^^^^^[0m
[0m2021.07.19 22:26:15 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:31: stale bloop error: not found: value name
     var displyName = println(name)
                              ^^^^[0m
[0m2021.07.19 22:26:15 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:23:28: stale bloop error: not found: value age
   var displyAge = println(age)
                           ^^^[0m
Jul 19, 2021 10:26:22 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:26:22 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:26:24 INFO  time: compiled vs-code-example in 1.71s[0m
[0m2021.07.19 22:26:36 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 37m 51.758s)[0m
[0m2021.07.19 22:26:42 INFO  Listening for transport dt_socket at address: 49692[0m
[0m2021.07.19 22:26:44 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:26:48 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:26:48 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:26:48 INFO  Trying to attach to remote debuggee VM 127.0.0.1:49692 .[0m
[0m2021.07.19 22:26:48 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.19 22:26:48 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.19 22:29:01 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 40m 16.374s)[0m
[0m2021.07.19 22:29:01 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:29:01 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:29:02 INFO  time: compiled vs-code-example in 1.15s[0m
[0m2021.07.19 22:29:09 INFO  Listening for transport dt_socket at address: 51823[0m
[0m2021.07.19 22:29:10 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:29:14 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:29:14 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:29:14 INFO  Trying to attach to remote debuggee VM 127.0.0.1:51823 .[0m
[0m2021.07.19 22:29:14 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.19 22:29:36 INFO  Enter your name:  Enter your age: [0m
[0m2021.07.19 22:29:36 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.19 22:34:32 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 45m 47.019s)[0m
[0m2021.07.19 22:34:32 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:34:32 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:34:32 INFO  time: compiled vs-code-example in 0.29s[0m
[0m2021.07.19 22:34:34 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:34:34 INFO  time: compiled vs-code-example in 0.27s[0m
Jul 19, 2021 10:34:36 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:35:20 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:30: stale bloop error: too many arguments for method display: ()Unit
     println("Your name: " + display(name))
                             ^^^^^^^^^^^^^[0m
[0m2021.07.19 22:35:20 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:30: stale bloop error: too many arguments for method display: ()Unit
     println("Your name: " + display(name))
                             ^^^^^^^^^^^^^[0m
[0m2021.07.19 22:35:24 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:30: stale bloop error: too many arguments for method display: ()Unit
     println("Your name: " + display(name))
                             ^^^^^^^^^^^^^[0m
[0m2021.07.19 22:35:24 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:30: stale bloop error: too many arguments for method display: ()Unit
     println("Your name: " + display(name))
                             ^^^^^^^^^^^^^[0m
Jul 19, 2021 10:35:56 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:35:56 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:35:57 INFO  time: compiled vs-code-example in 1.05s[0m
Jul 19, 2021 10:36:11 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassNotFoundException: proj0
java.util.concurrent.CompletionException: java.lang.ClassNotFoundException: proj0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: proj0
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.$anonfun$findClassAndBuildTarget$1(BuildTargetClassesFinder.scala:107)
	at scala.Option.fold(Option.scala:251)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findClassAndBuildTarget(BuildTargetClassesFinder.scala:108)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findMainClassAndItsBuildTarget(BuildTargetClassesFinder.scala:36)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$1(DebugProvider.scala:328)
	at scala.meta.internal.metals.debug.DebugProvider$$anonfun$withRebuildRetry$1.$anonfun$applyOrElse$5(DebugProvider.scala:560)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:36:41 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:36:41 INFO  time: compiled vs-code-example in 0.23s[0m
[0m2021.07.19 22:36:44 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:36:44 INFO  time: compiled vs-code-example in 0.21s[0m
Jul 19, 2021 10:36:46 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassNotFoundException: proj0
java.util.concurrent.CompletionException: java.lang.ClassNotFoundException: proj0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: proj0
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.$anonfun$findClassAndBuildTarget$1(BuildTargetClassesFinder.scala:107)
	at scala.Option.fold(Option.scala:251)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findClassAndBuildTarget(BuildTargetClassesFinder.scala:108)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findMainClassAndItsBuildTarget(BuildTargetClassesFinder.scala:36)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$1(DebugProvider.scala:328)
	at scala.meta.internal.metals.debug.DebugProvider$$anonfun$withRebuildRetry$1.$anonfun$applyOrElse$5(DebugProvider.scala:560)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 22:37:01 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:37:01 INFO  time: compiled vs-code-example in 0.94s[0m
[0m2021.07.19 22:37:11 INFO  Listening for transport dt_socket at address: 51326[0m
[0m2021.07.19 22:37:13 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:37:16 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:37:16 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:37:16 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.19 22:37:58 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:37:58 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 49m 12.895s)[0m
[0m2021.07.19 22:37:58 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:37:58 INFO  time: compiled vs-code-example in 0.94s[0m
[0m2021.07.19 22:38:05 INFO  Listening for transport dt_socket at address: 58037[0m
[0m2021.07.19 22:38:06 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:38:10 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:38:10 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:38:10 INFO  Canceling debug proxy for [proj0][0m
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 606, 606, 619)
[0m2021.07.19 22:39:09 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:39:09 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 50m 24.029s)[0m
[0m2021.07.19 22:39:09 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:39:09 INFO  time: compiled vs-code-example in 0.95s[0m
[0m2021.07.19 22:39:17 INFO  Listening for transport dt_socket at address: 58048[0m
[0m2021.07.19 22:39:18 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:39:21 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:39:21 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:40:05 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:40:05 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 51m 20.33s)[0m
[0m2021.07.19 22:40:05 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:40:05 INFO  time: compiled vs-code-example in 0.95s[0m
[0m2021.07.19 22:40:13 INFO  Listening for transport dt_socket at address: 53830[0m
[0m2021.07.19 22:40:14 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:40:17 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:40:17 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:40:17 INFO  Trying to attach to remote debuggee VM 127.0.0.1:53830 .[0m
[0m2021.07.19 22:40:17 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.19 22:41:10 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 52m 24.908s)[0m
[0m2021.07.19 22:41:10 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:41:10 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:41:11 INFO  time: compiled vs-code-example in 1.18s[0m
[0m2021.07.19 22:41:18 INFO  Listening for transport dt_socket at address: 63875[0m
[0m2021.07.19 22:41:19 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:41:23 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:41:23 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:41:23 INFO  Trying to attach to remote debuggee VM 127.0.0.1:63875 .[0m
[0m2021.07.19 22:41:23 INFO  Attaching to debuggee VM succeeded.[0m
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 606, 606, 619)
[0m2021.07.19 22:42:16 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 53m 31.409s)[0m
[0m2021.07.19 22:42:16 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:42:16 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:42:16 INFO  time: compiled vs-code-example in 0.86s[0m
[0m2021.07.19 22:42:24 INFO  Listening for transport dt_socket at address: 54368[0m
[0m2021.07.19 22:42:25 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:42:29 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:42:29 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:42:29 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54368 .[0m
[0m2021.07.19 22:42:29 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.19 22:43:34 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 54m 49.49s)[0m
[0m2021.07.19 22:43:34 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:43:34 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:43:34 INFO  time: compiled vs-code-example in 0.82s[0m
[0m2021.07.19 22:43:42 INFO  Listening for transport dt_socket at address: 54384[0m
[0m2021.07.19 22:43:43 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:43:47 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:43:47 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:43:47 INFO  Trying to attach to remote debuggee VM 127.0.0.1:54384 .[0m
[0m2021.07.19 22:43:47 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.19 22:49:58 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:49:58 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 1h 1m 13.751s)[0m
[0m2021.07.19 22:49:58 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:50:00 INFO  time: compiled vs-code-example in 1.75s[0m
[0m2021.07.19 22:50:07 INFO  Listening for transport dt_socket at address: 59171[0m
[0m2021.07.19 22:50:09 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.19 22:50:12 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.19 22:50:12 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.19 22:50:12 INFO  Trying to attach to remote debuggee VM 127.0.0.1:59171 .[0m
[0m2021.07.19 22:50:12 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.19 22:50:46 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:50:46 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 1h 2m 1.337s)[0m
[0m2021.07.19 22:50:46 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:50:46 INFO  time: compiled vs-code-example in 0.25s[0m
[0m2021.07.19 22:50:48 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 22:50:48 INFO  time: compiled vs-code-example in 0.31s[0m
Jul 19, 2021 10:50:51 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

Jul 19, 2021 11:07:27 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.19 23:07:28 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.19 23:07:30 INFO  time: compiled vs-code-example in 1.98s[0m
No more data in the server stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.20 09:15:17 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 09:15:24 INFO  Canceling debug proxy for [proj0][0m
No more data in the client stdin, exiting...
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 36448'...
Waiting for the bsp connection to come up...
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 31197'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:31197...
The server is listening for incoming connections at tcp://127.0.0.1:31197...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:36448...
The server is listening for incoming connections at tcp://127.0.0.1:36448...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 09:17:38 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 09:17:38 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 09:17:38 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.20 09:17:38 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
No more data in the server stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.20 09:17:38 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:17:40 INFO  time: compiled vs-code-example in 1.95s[0m
[0m2021.07.20 09:20:32 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 2m 54.499s)[0m
[0m2021.07.20 09:20:32 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:20:32 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:20:32 INFO  time: compiled vs-code-example in 0.31s[0m
[0m2021.07.20 09:20:35 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:20:35 INFO  time: compiled vs-code-example in 0.27s[0m
Jul 20, 2021 9:20:37 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:23:15 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 5m 36.894s)[0m
[0m2021.07.20 09:23:15 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:23:15 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:23:15 INFO  time: compiled vs-code-example in 0.24s[0m
[0m2021.07.20 09:23:17 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:23:17 INFO  time: compiled vs-code-example in 0.23s[0m
Jul 20, 2021 9:23:19 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:24:42 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:42 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:42 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:42 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:42 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:42 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:47 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:47 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:47 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:46 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:47 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:47 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:47 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:48 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:48 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:48 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:49 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:48 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:48 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:48 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:50 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:50 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:50 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:50 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:50 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:50 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:51 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:54 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:54 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:54 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:24: stale bloop error: not found: value monthy_expenses
     var DI = DI_ratio(monthy_expenses,monthly_income)
                       ^^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:21:40: stale bloop error: not found: value monthly_income
     var DI = DI_ratio(monthy_expenses,monthly_income)
                                       ^^^^^^^^^^^^^^[0m
[0m2021.07.20 09:24:53 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:22:14: stale bloop error: missing argument list for method main in object proj0
Unapplied methods are only converted to functions when a function type is expected.
You can make this conversion explicit by writing `main _` or `main(_)` instead of `main`.
     println(main)
             ^^^^[0m
[0m2021.07.20 09:25:13 INFO  compiling vs-code-example (1 scala source)[0m
Jul 20, 2021 9:25:13 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:25:13 INFO  time: compiled vs-code-example in 0.5s[0m
Jul 20, 2021 9:26:40 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:26:40 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:26:40 INFO  time: compiled vs-code-example in 0.49s[0m
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 67, 67, 80)
[0m2021.07.20 09:28:27 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 10m 49.57s)[0m
[0m2021.07.20 09:28:27 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:28:27 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:28:27 INFO  time: compiled vs-code-example in 0.35s[0m
[0m2021.07.20 09:28:30 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:28:30 INFO  time: compiled vs-code-example in 0.32s[0m
Jul 20, 2021 9:28:32 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:28:44 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 11m 6.265s)[0m
[0m2021.07.20 09:28:44 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:28:44 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:28:44 INFO  time: compiled vs-code-example in 0.29s[0m
[0m2021.07.20 09:28:47 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:28:47 INFO  time: compiled vs-code-example in 0.33s[0m
Jul 20, 2021 9:28:49 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

Jul 20, 2021 9:29:52 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 3257
Jul 20, 2021 9:31:03 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:31:03 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:31:03 INFO  time: compiled vs-code-example in 0.34s[0m
[0m2021.07.20 09:31:57 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 14m 19.575s)[0m
[0m2021.07.20 09:31:57 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:31:57 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:31:57 INFO  time: compiled vs-code-example in 0.33s[0m
[0m2021.07.20 09:32:00 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:32:00 INFO  time: compiled vs-code-example in 0.3s[0m
Jul 20, 2021 9:32:02 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

Jul 20, 2021 9:33:06 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:33:06 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:33:07 INFO  time: compiled vs-code-example in 1.22s[0m
Jul 20, 2021 9:33:37 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
java.util.concurrent.CompletionException: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.debug.WorkspaceErrorsException$: Cannot run class, since the workspace has errors.
	at scala.meta.internal.metals.debug.WorkspaceErrorsException$.<clinit>(DebugProvider.scala)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$2(DebugProvider.scala:332)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:33:37 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:33:37 INFO  time: compiled vs-code-example in 0.32s[0m
[0m2021.07.20 09:36:18 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:18 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:19 INFO  C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala:27:33: stale bloop error: recursive method branch needs result type
     println("DOI ratio is: " + branch())
                                ^^^^^^[0m
[0m2021.07.20 09:36:28 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 18m 49.675s)[0m
[0m2021.07.20 09:36:28 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:36:28 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:36:29 INFO  time: compiled vs-code-example in 1.09s[0m
[0m2021.07.20 09:36:33 INFO  Listening for transport dt_socket at address: 50837[0m
[0m2021.07.20 09:36:34 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.20 09:36:35 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.20 09:36:35 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.20 09:36:35 INFO  Trying to attach to remote debuggee VM 127.0.0.1:50837 .[0m
[0m2021.07.20 09:36:35 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.20 09:36:35 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 09:37:55 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 20m 17.091s)[0m
[0m2021.07.20 09:37:55 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:37:55 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:37:55 INFO  time: compiled vs-code-example in 0.77s[0m
[0m2021.07.20 09:38:00 INFO  Listening for transport dt_socket at address: 61853[0m
[0m2021.07.20 09:38:01 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.20 09:38:02 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.20 09:38:02 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.20 09:38:02 INFO  Trying to attach to remote debuggee VM 127.0.0.1:61853 .[0m
[0m2021.07.20 09:38:02 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.20 09:38:02 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 09:46:42 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 29m 4.235s)[0m
[0m2021.07.20 09:46:42 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:46:42 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:46:42 INFO  time: compiled vs-code-example in 0.31s[0m
[0m2021.07.20 09:46:45 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:46:45 INFO  time: compiled vs-code-example in 0.26s[0m
Jul 20, 2021 9:46:47 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:47:00 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 29m 22.034s)[0m
[0m2021.07.20 09:47:00 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:47:00 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:47:00 INFO  time: compiled vs-code-example in 0.22s[0m
[0m2021.07.20 09:47:02 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:47:02 INFO  time: compiled vs-code-example in 0.21s[0m
Jul 20, 2021 9:47:04 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:47:54 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 30m 16.301s)[0m
[0m2021.07.20 09:47:54 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:47:54 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:47:55 INFO  time: compiled vs-code-example in 1.1s[0m
[0m2021.07.20 09:48:00 INFO  Listening for transport dt_socket at address: 59773[0m
[0m2021.07.20 09:48:01 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.20 09:48:02 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.20 09:48:02 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.20 09:48:02 INFO  Trying to attach to remote debuggee VM 127.0.0.1:59773 .[0m
[0m2021.07.20 09:48:02 INFO  Attaching to debuggee VM succeeded.[0m
[0m2021.07.20 09:48:02 ERROR Error: Main method not found in class proj0, please define the main method as:[0m
[0m2021.07.20 09:48:02 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 09:48:02 ERROR    public static void main(String[] args)[0m
[0m2021.07.20 09:48:02 ERROR or a JavaFX application class must extend javafx.application.Application[0m
[0m2021.07.20 09:49:52 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:49:52 INFO  time: compiled vs-code-example in 0.81s[0m
No more data in the client stdin, exiting...
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 25939'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:25939...
The server is listening for incoming connections at tcp://127.0.0.1:25939...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 09:49:56 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 09:55:09 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:55:09 INFO  time: compiled vs-code-example in 0.29s[0m
Jul 20, 2021 9:55:11 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassNotFoundException: proj0
java.util.concurrent.CompletionException: java.lang.ClassNotFoundException: proj0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: proj0
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.$anonfun$findClassAndBuildTarget$1(BuildTargetClassesFinder.scala:107)
	at scala.Option.fold(Option.scala:251)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findClassAndBuildTarget(BuildTargetClassesFinder.scala:108)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findMainClassAndItsBuildTarget(BuildTargetClassesFinder.scala:36)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$1(DebugProvider.scala:328)
	at scala.meta.internal.metals.debug.DebugProvider$$anonfun$withRebuildRetry$1.$anonfun$applyOrElse$5(DebugProvider.scala:560)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 09:56:14 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:56:14 INFO  time: compiled vs-code-example in 0.27s[0m
[0m2021.07.20 09:56:16 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 09:56:16 INFO  time: compiled vs-code-example in 0.21s[0m
Jul 20, 2021 9:56:19 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassNotFoundException: proj0
java.util.concurrent.CompletionException: java.lang.ClassNotFoundException: proj0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: proj0
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.$anonfun$findClassAndBuildTarget$1(BuildTargetClassesFinder.scala:107)
	at scala.Option.fold(Option.scala:251)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findClassAndBuildTarget(BuildTargetClassesFinder.scala:108)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findMainClassAndItsBuildTarget(BuildTargetClassesFinder.scala:36)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$1(DebugProvider.scala:328)
	at scala.meta.internal.metals.debug.DebugProvider$$anonfun$withRebuildRetry$1.$anonfun$applyOrElse$5(DebugProvider.scala:560)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 11:03:17 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:03:17 INFO  time: compiled vs-code-example in 0.29s[0m
Jul 20, 2021 11:03:20 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassNotFoundException: proj0
java.util.concurrent.CompletionException: java.lang.ClassNotFoundException: proj0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: proj0
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.$anonfun$findClassAndBuildTarget$1(BuildTargetClassesFinder.scala:107)
	at scala.Option.fold(Option.scala:251)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findClassAndBuildTarget(BuildTargetClassesFinder.scala:108)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findMainClassAndItsBuildTarget(BuildTargetClassesFinder.scala:36)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$1(DebugProvider.scala:328)
	at scala.meta.internal.metals.debug.DebugProvider$$anonfun$withRebuildRetry$1.$anonfun$applyOrElse$5(DebugProvider.scala:560)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 11:04:24 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:04:24 INFO  time: compiled vs-code-example in 0.26s[0m
[0m2021.07.20 11:04:26 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:04:26 INFO  time: compiled vs-code-example in 0.25s[0m
Jul 20, 2021 11:04:29 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassNotFoundException: proj0
java.util.concurrent.CompletionException: java.lang.ClassNotFoundException: proj0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:42)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: proj0
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.$anonfun$findClassAndBuildTarget$1(BuildTargetClassesFinder.scala:107)
	at scala.Option.fold(Option.scala:251)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findClassAndBuildTarget(BuildTargetClassesFinder.scala:108)
	at scala.meta.internal.metals.debug.BuildTargetClassesFinder.findMainClassAndItsBuildTarget(BuildTargetClassesFinder.scala:36)
	at scala.meta.internal.metals.debug.DebugProvider.$anonfun$resolveMainClassParams$1(DebugProvider.scala:328)
	at scala.meta.internal.metals.debug.DebugProvider$$anonfun$withRebuildRetry$1.$anonfun$applyOrElse$5(DebugProvider.scala:560)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 11:14:51 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:14:53 INFO  time: compiled vs-code-example in 2.25s[0m
[0m2021.07.20 11:15:01 INFO  Listening for transport dt_socket at address: 59710[0m
Jul 20, 2021 11:15:07 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 4582
[0m2021.07.20 11:15:07 INFO  time: code lens generation in 1.92s[0m
[0m2021.07.20 11:15:07 INFO  tracing is disabled for protocol dap-server, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-server.trace.json[0m
[0m2021.07.20 11:15:10 INFO  tracing is disabled for protocol dap-client, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\dap-client.trace.json[0m
[0m2021.07.20 11:15:10 INFO  Starting debug proxy for [proj0][0m
[0m2021.07.20 11:15:10 INFO  Trying to attach to remote debuggee VM 127.0.0.1:59710 .[0m
[0m2021.07.20 11:15:10 INFO  Attaching to debuggee VM succeeded.[0m
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 59, 59, 72)
[0m2021.07.20 11:25:17 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 11:25:17 ERROR Read data from io exception: java.net.SocketException: Socket closed[0m
[0m2021.07.20 11:25:32 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 11:25:32 ERROR Read data from io exception: java.net.SocketException: Socket closed[0m
[0m2021.07.20 11:25:39 ERROR Read data from io exception: java.net.SocketException: Socket closed[0m
[0m2021.07.20 11:25:39 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 11:25:57 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 11:25:57 INFO  Enter your name:  [0m
[0m2021.07.20 11:25:57 INFO  shutting down Metals[0m
[0m2021.07.20 11:25:58 INFO  Shut down connection with build server.[0m
[0m2021.07.20 11:25:57 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
Unexpected exception when forwarding server stdin ---> client stdout
No more data in the client stdin, exiting...
[0m2021.07.20 11:25:57 INFO  Canceling debug proxy for [proj0][0m
[0m2021.07.20 11:26:30 INFO  Started: Metals version 0.10.5 in workspace 'C:\Users\14694\OneDrive\Desktop\vs Code Example' for client Visual Studio Code 1.58.2.[0m
[0m2021.07.20 11:26:31 INFO  time: initialize in 1.07s[0m
[0m2021.07.20 11:26:33 INFO  Attempting to connect to the build server...[0m
[0m2021.07.20 11:26:33 WARN  no build target for: C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala[0m
[0m2021.07.20 11:26:33 INFO  skipping build import with status 'Installed'[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 26300'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:26300...
The server is listening for incoming connections at tcp://127.0.0.1:26300...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 11:26:38 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 11:26:38 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 43144'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:43144...
The server is listening for incoming connections at tcp://127.0.0.1:43144...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 11:26:42 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 11:26:42 INFO  time: Connected to build server in 9.2s[0m
[0m2021.07.20 11:26:42 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.20 11:26:43 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.07.20 11:26:43 INFO  time: Imported build in 0.22s[0m
[0m2021.07.20 11:26:44 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.07.20 11:26:48 INFO  time: code lens generation in 15s[0m
[0m2021.07.20 11:26:54 INFO  time: indexed workspace in 10s[0m
Jul 20, 2021 11:29:02 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
WARNING: Unmatched cancel notification for request id 29
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 59, 59, 72)
something's wrong: no file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala in Array[String]RangePosition(file:///C:/Users/14694/OneDrive/Desktop/vs%20Code%20Example/src/main/scala/proj0.scala, 59, 59, 72)
Jul 20, 2021 11:39:36 AM scala.meta.internal.pc.CompilerAccess scala$meta$internal$pc$CompilerAccess$$handleError
SEVERE: String index out of range: -1
java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.<init>(String.java:196)
	at scala.tools.nsc.interactive.Global.typeCompletions$1(Global.scala:1229)
	at scala.tools.nsc.interactive.Global.completionsAt(Global.scala:1252)
	at scala.meta.internal.pc.SignatureHelpProvider$$anonfun$8.apply(SignatureHelpProvider.scala:375)
	at scala.meta.internal.pc.SignatureHelpProvider$$anonfun$8.apply(SignatureHelpProvider.scala:373)
	at scala.Option.map(Option.scala:146)
	at scala.meta.internal.pc.SignatureHelpProvider.treeSymbol(SignatureHelpProvider.scala:373)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCall$.unapply(SignatureHelpProvider.scala:198)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.visit(SignatureHelpProvider.scala:309)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.traverse(SignatureHelpProvider.scala:303)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.fromTree(SignatureHelpProvider.scala:272)
	at scala.meta.internal.pc.SignatureHelpProvider.signatureHelp(SignatureHelpProvider.scala:27)

[0m2021.07.20 11:49:52 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:49:52 INFO  Deduplicating compilation of vs-code-example from bsp client 'Metals 0.10.5' (since 23m 13.832s)[0m
[0m2021.07.20 11:49:52 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:49:53 INFO  time: compiled vs-code-example in 1.18s[0m
[0m2021.07.20 11:49:55 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:49:56 INFO  time: compiled vs-code-example in 1.03s[0m
Jul 20, 2021 11:49:58 AM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
java.util.concurrent.CompletionException: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:661)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:646)
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:29)
	at scala.concurrent.java8.FuturesConvertersImpl$CF.apply(FutureConvertersImpl.scala:26)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at scala.concurrent.BatchingExecutor$Batch.processBatch$1(BatchingExecutor.scala:67)
	at scala.concurrent.BatchingExecutor$Batch.$anonfun$run$1(BatchingExecutor.scala:82)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at scala.concurrent.BatchingExecutor$Batch.run(BatchingExecutor.scala:59)
	at scala.concurrent.Future$InternalCallbackExecutor$.unbatchedExecute(Future.scala:875)
	at scala.concurrent.BatchingExecutor.execute(BatchingExecutor.scala:110)
	at scala.concurrent.BatchingExecutor.execute$(BatchingExecutor.scala:107)
	at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:873)
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:72)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:288)
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:288)
	at scala.concurrent.Promise.complete(Promise.scala:53)
	at scala.concurrent.Promise.complete$(Promise.scala:52)
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:187)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: scala.meta.internal.metals.MetalsBspException: BSP connection failed in the attempt to get: DebugSessionAddress.  Compilation not successful
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:237)
	at scala.meta.internal.metals.BuildServerConnection$$anonfun$1.applyOrElse(BuildServerConnection.scala:229)
	at scala.concurrent.Future.$anonfun$recoverWith$1(Future.scala:417)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	... 4 more

[0m2021.07.20 11:51:16 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:51:18 INFO  time: compiled vs-code-example in 2.7s[0m
[0m2021.07.20 11:54:56 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 11:54:59 INFO  time: compiled vs-code-example in 2.76s[0m
[0m2021.07.20 12:16:35 INFO  shutting down Metals[0m
[0m2021.07.20 12:16:36 INFO  Shut down connection with build server.[0m
[0m2021.07.20 12:16:36 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
No more data in the server stdin, exiting...
No more data in the client stdin, exiting...
[0m2021.07.20 12:18:20 INFO  Started: Metals version 0.10.5 in workspace 'C:\Users\14694\OneDrive\Desktop\vs Code Example' for client Visual Studio Code 1.58.2.[0m
[0m2021.07.20 12:18:21 INFO  time: initialize in 0.55s[0m
[0m2021.07.20 12:18:22 INFO  Attempting to connect to the build server...[0m
[0m2021.07.20 12:18:21 WARN  no build target for: C:\Users\14694\OneDrive\Desktop\vs Code Example\src\main\scala\proj0.scala[0m
[0m2021.07.20 12:18:21 INFO  skipping build import with status 'Installed'[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 14414'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:14414...
The server is listening for incoming connections at tcp://127.0.0.1:14414...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 12:18:24 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 12:18:24 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 44712'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:44712...
The server is listening for incoming connections at tcp://127.0.0.1:44712...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 12:18:26 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 12:18:26 INFO  time: Connected to build server in 3.86s[0m
[0m2021.07.20 12:18:26 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.20 12:18:27 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.07.20 12:18:27 INFO  time: Imported build in 0.11s[0m
[0m2021.07.20 12:18:28 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.07.20 12:18:30 INFO  time: code lens generation in 7.96s[0m
[0m2021.07.20 12:18:34 INFO  time: indexed workspace in 6.86s[0m
[0m2021.07.20 12:45:46 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 12:45:48 INFO  time: compiled vs-code-example in 1.26s[0m
Jul 20, 2021 12:57:37 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
[0m2021.07.20 12:58:23 INFO  shutting down Metals[0m
[0m2021.07.20 12:58:23 INFO  Shut down connection with build server.[0m
[0m2021.07.20 12:58:23 INFO  Shut down connection with build server.[0m
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
No more data in the server stdin, exiting...
No more data in the client stdin, exiting...
No more data in the server stdin, exiting...
[0m2021.07.20 12:58:40 INFO  Started: Metals version 0.10.5 in workspace 'C:\Users\14694\OneDrive\Desktop\vs Code Example' for client Visual Studio Code 1.58.2.[0m
[0m2021.07.20 12:58:40 INFO  time: initialize in 0.38s[0m
[0m2021.07.20 12:58:41 INFO  Attempting to connect to the build server...[0m
[0m2021.07.20 12:58:40 INFO  skipping build import with status 'Installed'[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 43461'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:43461...
The server is listening for incoming connections at tcp://127.0.0.1:43461...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 12:58:42 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 12:58:42 INFO  Attempting to connect to the build server...[0m
Starting the bsp launcher for bloop...
Opening a bsp server connection with 'bsp --protocol tcp --port 33132'...
Waiting for the bsp connection to come up...
[0m[32m[D][0m Loading workspace settings from bloop.settings.json
[0m[32m[D][0m Waiting for a connection at 127.0.0.1:33132...
The server is listening for incoming connections at tcp://127.0.0.1:33132...
Starting thread that pumps stdin and redirects it to the bsp server...
Starting thread that pumps server stdout and redirects it to the client stdout...
[0m2021.07.20 12:58:44 INFO  tracing is disabled for protocol BSP, to enable tracing of incoming and outgoing JSON messages create an empty file at C:\Users\14694\AppData\Local\scalameta\metals\cache\bsp.trace.json[0m
[0m2021.07.20 12:58:44 INFO  time: Connected to build server in 2.72s[0m
[0m2021.07.20 12:58:44 INFO  Connected to Build server: Bloop v1.4.8-19-4d9f966b[0m
[0m2021.07.20 12:58:44 INFO  time: Imported build in 0.12s[0m
[0m2021.07.20 12:58:47 INFO  time: indexed workspace in 3.37s[0m
[0m2021.07.20 14:40:57 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 14:41:00 INFO  time: compiled vs-code-example in 2.05s[0m
[0m2021.07.20 15:49:20 INFO  compiling vs-code-example (1 scala source)[0m
[0m2021.07.20 15:49:21 INFO  time: compiled vs-code-example in 1.26s[0m
